jumpscale/clients/stellar/__init__.py

```diff
@@ -7,9 +7,16 @@
 from .stellar import Stellar
 
 ACTIVATION_WALLET_ADDRESS = "GCKLGWHEYT2V63HC2VDJRDWEY3G54YSHHPOA6Q3HAPQUGA5OZDWZL7KW"
+TRANSACTION_FEES = 0.01
 
 
 class StellarFactory(StoredFactory):
+    @property
+    def TRANSACTION_FEES(self):
```

Comment:
actually, let's remove the property and import TRANSACTION_FEES from the the module. sorry for the confusion